q2_fondue/types/_format.py

```diff
@@ -100,11 +100,27 @@ def _validate_id(self, _id: str):
 
     def _validate_(self, level):
         df = pd.read_csv(str(self), sep='\t')
+        cols_df = df.columns.tolist()
```

Comment:
```suggestion
        cols = df.columns.tolist()
```

`cols_df` suggests that the variable is a DataFrame.